package.json

```diff
@@ -1,23 +1,4 @@
 {
-  "name": "performance",
-  "version": "1.0.0-beta.2",
-  "private": true,
-  "description": "Performance plugin from the WordPress Performance Group, which is a collection of standalone performance modules.",
-  "author": "The WordPress Contributors",
-  "license": "GPL-2.0-or-later",
-  "keywords": [
-    "performance",
-    "wordpress"
-  ],
-  "homepage": "https://wordpress.org/plugins/performance-lab/",
-  "repository": "git+https://github.com/WordPress/performance.git",
-  "bugs": {
-    "url": "https://github.com/WordPress/performance/issues"
-  },
-  "engines": {
-    "node": ">=10.0.0",
-    "npm": ">=6.9.0"
-  },
```

Comment:
Is it okay to remove this? I'd say it's useful information to specific which Node and NPM versions are required?

Reply:
@felixarntz IMO the exact idea of Node and NPM version can be extracted from configuration files i.e. `.npmrc` and `.nvmrc`. AFAIK this is useful when you are reading metadata from `package.json` itself. 

If you have come around Heroku, then you would have noticed that, when you deploy a Heroku app, it chooses the Node buildpack according to the `engines` metadata in `package.json`.

In our use case `.nvmrc` and `.npmrc` should be enough.

Reply:
Thanks @thelovekesh, makes sense.